src/lib/features/feature-lifecycle/feature-lifecycle.e2e.test.ts

```diff
@@ -98,7 +98,8 @@ const uncompleteFeature = async (featureName: string, expectedCode = 200) => {
 function reachedStage(feature: string, stage: StageName) {
     return new Promise((resolve) =>
         featureLifecycleService.on(STAGE_ENTERED, (event) => {
-            if (event.stage === stage) resolve(stage);
+            if (event.stage === stage && event.feature === feature)
```

Comment:
when you introduce feature to a method signature that would be cool if you used it :)

Reply:
wow! I totally missed it!